### DISCUSSION TITLE
OSS projects willing to be in the list

### DISCUSSION THREAD

**Discussion by kdy1** [swc-project/swc-ecosystem-ci#3] [Q&A]
If you are willing to make your OSS project join here, please left a link to the project.

**Comment by thepassle** [comment]
Seeing as Rollup recently moved to using SWC's parser in V4, I think it could be good to include. 
https://rollupjs.org/

Disclaimer: I'm not a maintainer of Rollup

**Comment by ArnaudBarre** [comment]
Hi 👋
I'm using a very thin party of the JS API in https://github.com/vitejs/vite-plugin-react-swc and the test suite is quite small but this will at least include testing few plugins (styled-component/emotion) and basic e2e Fast Refresh transformation.
